builder/js/ncf.js

```diff
@@ -1133,12 +1134,13 @@ $scope.onImportFileChange = function (fileEl) {
   // selected technique is correct if:
   // * There is at least one method call
   $scope.checkSelectedTechnique = function() {
+    if($scope.selectedTechnique===undefined || $scope.selectedTechnique===undefined) return false;
```

Comment:
You check the same value twice, remove second $scope.selectedTechnique===undefined

Reply:
My bad, I would probably want to check if it is striclty different from null too